### PR TITLE
feat(component): add primary variant to badge

### PR DESCRIPTION
**Pull request by changamanda**
Adds a primary variant to the badge

Screen Shot 2021-04-13 at 4 24 22 PM

### CODE REVIEW

**Comment by changamanda** [comment]
Looks like I don't have permissions to add reviewers/assignee/labels

File: packages/big-design/src/components/Badge/spec.tsx [hunk 1]
```diff
@@ -51,6 +51,13 @@ test('render secondary Badge', () => {
   expect(container.firstChild).toHaveStyle(`background-color: ${theme.colors.secondary60}`);
 });
 
+test('render info Badge', () => {
```
Comment (chanceaclark) [inline packages/big-design/src/components/Badge/spec.tsx:54]:
```suggestion
test('render primary Badge', () => {
```

**Review by chanceaclark — COMMENTED** [review]
Small change, but looks good!

**Review by jorgemoya — APPROVED** [review]
Looks great I just would address the one comment. 💯

**Review by jordan-massingill — APPROVED** [review]
👍

**Review by deini — APPROVED** [review]

**Pull request merged**